test/functional/http/get/delegates.js

```diff
@@ -255,6 +255,18 @@ describe('GET /delegates', () => {
 				});
 			});
 
+			it('using numeric string should be ok', () => {
+				return delegatesEndpoint.makeRequest(
```

Comment:
Would be nice if we actually checked that genesis account is returned in the response.

Reply:
The test is related to the request itself not to the response. Anyway, the result is 0 because it is not the genesis address what it allows the fuzzy search. It only allows usernames.

Reply:
New test added.